packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -183,12 +206,32 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
             lastEvidence = next;
           }
 
-          // sufficient agreement, so remove from pending and publish
+          // take the union of risks identified from all operators
+          /**  @type {Set<string>} */
+          const setUnionRisks = new Set();
+          const riskStores = [...risks.values()].filter(store =>
+            store.has(txHash),
+          );
+          for (const store of riskStores) {
+            const next = store.get(txHash);
+            for (const risk of next.risksIdentified ?? []) {
+              setUnionRisks.add(risk);
+            }
+          }
+          const unionRisks = [...setUnionRisks.values()].sort();
```

Comment:
please factor out into a helper function. a unit test would be nice. Also it should be clear that is just the string[] object, not a union of objects shaped like state.risk. Best to use `risksIdentified` to shape of the object being passed to `publish`.
```suggestion
          const risksIdentifier = allRisksIdentified(risks);
```

Reply:
Just pulled some of it out into a helper where I could. I think the existing test in `packages/fast-usdc/test/exos/transaction-feed.test.ts` covers it pretty well but I added some more test cases.